woocommerce-services.php

```diff
@@ -338,8 +338,19 @@ function() {
 					}
 				}
 			);
-			add_action( 'plugins_loaded', array( $this, 'jetpack_on_plugins_loaded' ), 1 );
+
 			add_action( 'plugins_loaded', array( $this, 'on_plugins_loaded' ) );
+
+			if ( $this->are_woo_shipping_and_woo_tax_active() ) {
+				/**
+				 * Used to let Woo Shipping and Woo Tax know WCS&T will handle the plugins' coexistence
+				 * by displaying an appropriate notice and not registering its functionality.
+				 */
```

Comment:
[Non-blocking] If we're leaning up against the WP standards for `do_action` and `apply_filters` when adding hooks like this, then the summary should be 1 line and if it provides value with further explanation, then the description should have a new line between it and the summary 😄 

https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#4-hooks-actions-and-filters